src/state/predictions/helpers.ts

```diff
@@ -228,6 +244,25 @@ export const getMarketData = async (): Promise<{
   }
 }
 
+export const getTotalWon = async (): Promise<number> => {
+  const response = (await request(
+    GRAPH_API_PREDICTION,
+    gql`
+      query getTotalWonData {
+        market(id: 1) {
+          totalBNB
+          totalBNBTreasury
+        }
+        rounds(where: { position: ${BetPosition.HOUSE} }) {
+          totalAmount
+        }
```

Comment:
Pass in the variables like this with the second argument. You can check `getRound` as an example.

Reply:
Done